ephios/core/templatetags/user_extras.py

```diff
@@ -78,10 +78,9 @@ def intersects(a, b):
     return bool(set(a) & set(b))
 
 
-@register.filter(name="has_permission")
-def group_has_permission(group, permission):
-    app_label, codename = permission.split(".")
-    return group.permissions.filter(content_type__app_label=app_label, codename=codename).exists()
```

Comment:
this was not in use as far as I can see... Can you confirm? @jeriox

Reply:
cannot find anything either